src/backend/expungeservice/charges_summarizer.py

```diff
@@ -38,6 +38,8 @@ def build_charges_for_summary_panel(record: Record) -> ChargesForSummaryPanel:
     @staticmethod
     def _primary_sort(charge: Charge, record: Record):
         charge_eligibility = charge.expungement_result.charge_eligibility
+
+        charge_dict = {"nma":0,"ine":1,"nnn":2, "nni":3, "ncn":4, "nci":5, "fnn":6, "fcn":7, "fni":8, "fci":9 }
```

Comment:
Could we use an str, Enum and spell these out?

Reply:
I can spell out the different charges, but could you elaborate a bit on the benefits of enum vs. a regular dict for this? Thanks!

Reply:
It isn't that different - just a bit more robust. I also just prefer not to abbreviate. For example, I don't know what `fni` means.